clients/src/main/java/org/apache/kafka/clients/NodeApiVersions.java

```diff
@@ -73,6 +75,23 @@ public static NodeApiVersions create(Collection<ApiVersion> overrides) {
         return new NodeApiVersions(apiVersions);
     }
 
+    // Used for testing
```

Comment:
Let's add some JavaDoc for this function

Reply:
Done.